xmpp_muc.go

```diff
@@ -48,7 +48,7 @@ func (c *Client) JoinMUC(jid, nick string, history_type, history int, history_da
 	switch history_type {
 	case NoHistory:
 		return fmt.Fprintf(c.conn, "<presence to='%s/%s'>\n" +
-			"<x xmlns='%s' />\n" +
+			"<x xmlns='%s' /></x>\n" +
```

Comment:
seems to be wrong

Reply:
Right, strange the tests did not catch it yesterday. Tested it again with more MUCs to join, works.